packages/salesforcedx-utils-vscode/test/jest/helpers/paths.test.ts

```diff
@@ -70,4 +71,15 @@ describe('test project paths', () => {
       expect(relativeStateFolder).toEqual(FAKE_STATE_FOLDER);
     });
   });
+
+  describe('test relativeToolsFolder', () => {
+    it('should be defined', () => {
+      expect(projectPaths.relativeToolsFolder).toBeDefined();
+    });
+
+    it('should return a path to the relative tools folder', () => {
+      const relativeToolsFolder = projectPaths.relativeToolsFolder();
+      expect(relativeToolsFolder).toEqual(FAKE_TOOLS_FOLDER);
```

Comment:
So for this test you should be verifying that the value returned is the combo of the relative state folder and the tools constant.  I'd probably be ok with something like 
```      
expect(relativeToolsFolder).toEqual(
        path.join(FAKE_STATE_FOLDER, TOOLS) // TOOLS exported from paths.ts
      );
```

But even better would be to mock the projectPaths.relativeStateFolder() response for this test and have something like 
```
     relativeStateFolderMock.resolves(FAKE_STATE_FOLDER);
     const relativeToolsFolder = projectPaths.relativeToolsFolder();
      expect(relativeToolsFolder).toEqual(
        path.join(FAKE_STATE_FOLDER, TOOLS) // TOOLS exported from paths.ts
      );
```

Reply:
Looks like your test is failing with 
```
    Expected: ".sfdx/tools"     Received: ".sfdx\\tools"```
```
another good reason to update to the above suggested method for the assertion.